sleap/nn/tracking.py

```diff
@@ -97,7 +104,7 @@ class FlowCandidateMaker:
     of_window_size: int = 21
     of_max_levels: int = 3
 
-    save_shifted_instances: bool = False
+    save_shifted_instances: bool = True
```

Comment:
We'd also prefer not to change any defaults. In this case, it's possible that saving a significant number of matched instances may affect performance, so maybe we can leave it at `False` by default and still enable the functionality by manually setting when it's instantiated.

Reply:
Yes it makes sense, I reverted it